fastlane/metadata/android/en-US/short_description.txt

```diff
@@ -1 +1 @@
-Free and open-source 2FA app with encrypted backups and Wear OS support
```

Comment:
Actually, for `en-US` this is OK (71 byte altogether) and thus not required to be shortened. As pointed out in #660 it's rather `de-DE`, `pl-PL`, `pt-BR`, `ru-RU`, `sk-SK` and `uk-UA` which need to be shortened.

Reply:
https://github.com/jamie-mh/AuthenticatorPro/issues/660#issuecomment-1453921055